ScheduledStartandStopforSAPSystems/azuredeploy.json

```diff
@@ -240,446 +240,1370 @@
           },
           "triggers": "[if(equals(parameters('Frequency'), 'Month'), variables('monthlySchedule'), variables('otherSchedule'))]",
           "actions": {
-            "SAP_operation_type": {
-              "actions": {
-                "For_each_VIS_resource_start": {
-                  "actions": {
-                    "Database_start_operation": {
-                      "actions": {
-                        "For_each_DB_instance_start": {
-                          "actions": {
-                            "Initiate_database_instance_start": {
-                              "inputs": {
-                                "body": {
-                                  "startVm": "@parameters('IncludeVirtualMachineOperation')"
-                                },
-                                "host": {
-                                  "connection": {
-                                    "name": "@parameters('$connections')['arm']['connectionId']"
-                                  }
+            "Check_if_list_of_resources_with_failed_operation_empty": {
+                "actions": {},
+                "else": {
+                    "actions": {
+                        "Terminate": {
+                            "inputs": {
+                                "runError": {
+                                    "code": "The @{parameters('OperationName')} has failed on some resources. Please check the list of resource Ids in the variable - \"List of Resource IDs with failure\""

```

Comment:
Does
@(variables('List of Resource IDs with failure')) not work here ?

Reply:
No Ajay, I have cross-checked this.